list.go

```diff
@@ -1,30 +1,30 @@
 package main
 
 var (
-	licensesList = []struct {
-		key      string
+	licensesList = map[string]struct {
```

Comment:
Minor: rename this variable to `licenses`? As is, a variable named "xxxList" is assigned to a map value.

Reply:
I've moved the whole list variable to `license.go`, since this is the only place that it is referenced, and since the file name `list.go` is now just as redundant as the `licensesList` variable name.

https://github.com/yuri-norwood/license/blob/fc313c3e02fa31e5e944f6d6e08f9ca9be755a7c/license.go#L10